### PR TITLE
Add support for applying a hotfix on the Satellite6 Server.

### PR DESCRIPTION
**Pull request by kbidarkar**
The repos and the script should reside on the HTTP_SERVER_HOSTNAME
specified in the jenkins-config repo.

The repos and the script are unique depending upon the OS version.

### CODE REVIEW

**Comment by kbidarkar** [comment]
So, while working on this PR, I came across a situation where in initially I thought we should allow the user to specify the repo and script, but soon realized that it would lead to unnecessary parameter clutter on the UI. Hence added a help-tip for it.

File: automation_tools/__init__.py [hunk 1]
```diff
@@ -1358,6 +1358,34 @@ def configure_realm(admin_password=None, keytab_url=None, realm=None,
     run('service foreman-proxy restart')
 
 
+def apply_hotfix():
+    """Apply a hotfix for the Satellite6 Server
+
+    Expects the following environment variables:
+
+    HOTFIX
+        Specifies the choice for the hotfix, the default or custom.
```
Comment (sghai) [inline automation_tools/__init__.py]: So `Default` is when Hotfix is available through any repo and `custom` when its available through custom script ?

Comment (kbidarkar) [inline automation_tools/__init__.py]: Yes, that is right.

**Comment by sthirugn** [comment]
ACK

**Comment by sghai** [comment]
ACK

**Pull request merged**